### PR TITLE
Show access key and make better description for whitelist

### PR DESCRIPTION
**Pull request by OliverNeff**
## Description:
By review on 2018-01-18 we noticed that the access key is replaced with **** , but so you can't see the access key when you forgott it.
We decide that it is ok when it is shown in plane text to begin. 

## Improvements
- Access key is shown.

## Known Issues:
see #501 

------
_Remember to prefix your PR-Title with `⚠ WIP: ` if you still work on it.
If you have reached a final state, remove the prefix._

### CODE REVIEW

File: app/webFrontend/src/app/course/course-edit/general-tab/general-tab.component.html [hunk 1]
```diff
@@ -26,7 +26,8 @@
   </mat-form-field>
 
   <div class="csv-upload" *ngIf="enrollType === enrollTypeConstants.ENROLL_TYPE_WHITELIST">
-    <p>Upload your OBS csv file with your course members for whitelisting students.</p>
+    <p>Upload your OBS csv file with your course members for whitelisting students.
```
Comment (PatrickSkowronek) [inline app/webFrontend/src/app/course/course-edit/general-tab/general-tab.component.html]: This should not be in this pr!

File: app/webFrontend/src/app/course/course-edit/general-tab/general-tab.component.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ export class GeneralTabComponent implements OnInit {
         (val: any) => {
           this.course = val.name;
           this.description = val.description;
-          this.accessKey = val.hasAccessKey ? '****' : '';
+          this.accessKey = val.accessKey;
```
Comment (PatrickSkowronek) [inline app/webFrontend/src/app/course/course-edit/general-tab/general-tab.component.ts]: val.accessKey is undefined and the input field will be empty!

Comment (OliverNeff) [reply app/webFrontend/src/app/course/course-edit/general-tab/general-tab.component.ts]: Ohh sorry there is missing something in this merge request. ^^

File: app/webFrontend/src/app/course/course-edit/general-tab/general-tab.component.html (hunk 1 above)
Comment (OliverNeff) [reply app/webFrontend/src/app/course/course-edit/general-tab/general-tab.component.html]: You are right, it should be in: #493

File: app/webFrontend/src/app/course/course-edit/general-tab/general-tab.component.ts (hunk 2 above)
Comment (OliverNeff) [reply app/webFrontend/src/app/course/course-edit/general-tab/general-tab.component.ts]: It should be done with this: 
https://github.com/h-da/geli/pull/494/commits/0c69e326696abb3c10312db02d3252b6f092a041

**Pull request merged**